### ISSUE TITLE
LMS SigVer: Vector sets are not available (general exception)

### ISSUE THREAD

**Issue by AlexThurston** [usnistgov/ACVP-Server#282]
**environment**
Demo

**testSessionId**
433435

**vsId**
1799231

**Algorithm registration**
```
{
    "algorithm": "LMS",
    "revision": "1.0",
    "mode": "sigVer",
    "capabilities": {
        "lmsModes": [
            "LMS_SHA256_M24_H5",
            "LMS_SHA256_M24_H10",
            "LMS_SHA256_M24_H15",
            "LMS_SHA256_M24_H20",
            "LMS_SHA256_M24_H25",
            "LMS_SHA256_M32_H5",
            "LMS_SHA256_M32_H10",
            "LMS_SHA256_M32_H15",
            "LMS_SHA256_M32_H20",
            "LMS_SHA256_M32_H25",
            "LMS_SHAKE_M24_H5",
            "LMS_SHAKE_M24_H10",
            "LMS_SHAKE_M24_H15",
            "LMS_SHAKE_M24_H20",
            "LMS_SHAKE_M24_H25",
            "LMS_SHAKE_M32_H5",
            "LMS_SHAKE_M32_H10",
            "LMS_SHAKE_M32_H15",
            "LMS_SHAKE_M32_H20",
            "LMS_SHAKE_M32_H25"
        ],
        "lmOtsModes": [
            "LMOTS_SHA256_N24_W1",
            "LMOTS_SHA256_N24_W2",
            "LMOTS_SHA256_N24_W4",
            "LMOTS_SHA256_N24_W8",
            "LMOTS_SHA256_N32_W1",
            "LMOTS_SHA256_N32_W2",
            "LMOTS_SHA256_N32_W4",
            "LMOTS_SHA256_N32_W8",
            "LMOTS_SHAKE_N24_W1",
            "LMOTS_SHAKE_N24_W2",
            "LMOTS_SHAKE_N24_W4",
            "LMOTS_SHAKE_N24_W8",
            "LMOTS_SHAKE_N32_W1",
            "LMOTS_SHAKE_N32_W2",
            "LMOTS_SHAKE_N32_W4",
            "LMOTS_SHAKE_N32_W8"
        ]
    }
}
```

**Endpoint in which the error is experienced**
`https://demo.acvts.nist.gov/acvp/v1/testSessions/433435/vectorSets/1799231` GET

The serve is responding with:
```
[
  {
    "acvVersion": "1.0"
  },
  {
    "error": "General exception. Contact service provider."
  }
]
```

**Expected behavior**
I get vector sets

**Comment by jbrock24** [comment]
@AlexThurston Sorry, but it appears LMS testing is disabled why the fix is being implemented in response to: #279, we'll respond here when it's reenabled.

**Comment by livebe01** [comment]
This issue should now be resolved.
